Vagrantfile

```diff
@@ -50,6 +65,16 @@ Vagrant.configure(2) do |config|
     ubuntu.vm.network 'forwarded_port', guest: 1501, host: 1501   # site
     ubuntu.vm.network 'forwarded_port', guest: 8433, host: 8433   # Websockets
     ubuntu.vm.network 'forwarded_port', guest: 5432, host: 16432  # database
+    ubuntu.vm.provision 'shell', inline: $script
+  end
+
+  config.vm.define 'submitty-worker', autostart: autostart_worker do |ubuntu|
+    ubuntu.vm.box = 'bento/ubuntu-20.04'
+    # If this IP address changes, it must be changed in install_system.sh and 
+    # CONFIGURE_SUBMITTY.pyto allow the ssh connection
+    ubuntu.vm.network "private_network", ip: "172.18.2.8"
+    ubuntu.vm.network 'forwarded_port', guest: 22, host: 2220, id: 'ssh'
+    ubuntu.vm.provision 'shell', inline: $worker_script
   end
```

Comment:
Move this block after the `ubuntu-20.04` block. We should be grouping related blocks (`ubuntu-18.04` and `ubuntu-20.04` in this case) next to each other to enforce changes to one should be made to the other.

Reply:
Good idea, I'll group `ubuntu-20.04` and `ubuntu-18.04` together at the bottom since the `ubuntu-20.04` needs `submitty-worker` to already be provisioned to set up ssh keys for submitty_daemon.